src/equations/equations.jl

```diff
@@ -207,7 +207,20 @@ where `x` specifies the coordinates, `t` is the current time, and `equation` is
 struct BoundaryConditionNeumann{B}
     boundary_normal_flux_function::B
 end
+"""
+    NonConservativeLocal
+
+Struct used for multiple dispatch on non-conservative flux functions in the format of "local * symmetric". 
+When the argument nonconservative_type is of type `NonConservativeLocal`, the function returns the local part of the non-conservative term.
+"""
```

Comment:
```suggestion
When the argument `nonconservative_type` is of type `NonConservativeLocal`, 
the function returns the local part of the non-conservative term.
"""
```

Which function are you talking about? I don't see an argument `nonconservative_type` here

Reply:
It is an argument of the non-conservative flux functions that can be written as the product of local and symmetric parts. So far, I only implemented it for `flux_nonconservative_powell2`. See:
https://github.com/trixi-framework/Trixi.jl/blob/86884e81e7cfbe18d6f7390fc64d8d14f21bd574/src/equations/ideal_glm_mhd_2d.jl#L339-L461